video/version.h

```diff
@@ -0,0 +1,12 @@
+#ifndef VERSION_H
+#define VERSION_H
+
+#define		VERSION_MAJOR		2
+#define		VERSION_MINOR		0
+#define		VERSION_PATCH		0
+#define		VERSION_CANDIDATE	1			// Optional
+#define		VERSION_TYPE		"Alpha "	// RC, Alpha, Beta, etc.
```

Comment:
As per comments made against the equivalent PR in MOS AgonConsole8/agon-mos#16, the version number here is kinda a placeholder...

bumping to 2 is mostly about drawing a line in the sand from the old numbering system